tokio/src/stream/mod.rs

```diff
@@ -0,0 +1,93 @@
+//! Stream utilities for Tokio.
+//!
+//! `Stream`s are an asynchoronous version of standard library's Iterator.
+//!
```

Comment:
I think that ideally, there would be at least one example of using a `Stream`, even if it's minimal.